docs/content/reference/zero-trust/scoutsuite.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Scoutsuite"
+date: 2021-03-02T16:23:06+02:00
+draft: false
+---
+
+### What is ScoutSuite?
+
+<a href="https://github.com/nccgroup/ScoutSuite" target="_blank" >Scout Suite</a> is an open-source cloud security-auditing tool.
+It queries the cloud API to gather configuration data of the cloud infrastructure. Based on configuration
+data gathered ScoutSuite shows security issues and risks present in your infrastructure.
+
+### Which cloud providers are supported?
+
+So far the Infection Monkey only supports AWS.
```

Comment:
```suggestion
Today, the Infection Monkey only supports AWS.
```

Reply:
Fixed to `Currently`, as we refer to this point in time, not a particular time unit. Could be removed altogether though